cmd/crank/beta/render/cmd.go

```diff
@@ -45,6 +46,7 @@ type Cmd struct {
 	ContextFiles           map[string]string `mapsep:"," help:"Comma-separated context key-value pairs to pass to the Function pipeline. Values must be files containing JSON."`
 	ContextValues          map[string]string `mapsep:"," help:"Comma-separated context key-value pairs to pass to the Function pipeline. Values must be JSON. Keys take precedence over --context-files."`
 	IncludeFunctionResults bool              `short:"r" help:"Include informational and warning messages from Functions in the rendered output as resources of kind: Result."`
+	IncludeXRSpec          bool              `short:"x" help:"Include the XR spec to the rendered output."`
```

Comment:
```suggestion
	IncludeXRSpec          bool              `short:"x" help:"Include a direct copy of the input XR spec in the rendered output."`
```

CC @plumbis for wordsmithing suggestions. I want to clarify succinctly that the spec is copied verbatim from the input you give. Unlike the rest of the output we're not actually rendering or computing it in any way.

Reply:
I'd nix `direct` (what other kind of copy could it be?) but +1 on this suggestion.